core/commands/publish.go

```diff
@@ -102,7 +103,7 @@ Publish an <ipfs-path> to another public key (not implemented):
 	Marshalers: cmds.MarshalerMap{
 		cmds.Text: func(res cmds.Response) (io.Reader, error) {
 			v := res.Output().(*IpnsEntry)
-			s := fmt.Sprintf("Published name %s to %s\n", v.Name, v.Value)
+			s := fmt.Sprintf("Published %s to %s\n", v.Value, v.Name)
```

Comment:
this order is flipped from the input, if you add the optional entry (which today doesn't work), you'd get:

```
> ipfs name publish <name> <value>
Published <value> to <name>
```

this can be fine, but a heads up that it may be odd in the future.